torchgeo/datasets/nccm.py

```diff
@@ -0,0 +1,208 @@
+"""Northeastern China Crop Map Dataset."""
```

Comment:
All files need license information at the top of the file. You can copy this from any other file.